driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/unified/UnifiedCrudTest.java

```diff
@@ -37,6 +39,12 @@ public UnifiedCrudTest(final String fileDescription, final String testDescriptio
         super(schemaVersion, runOnRequirements, entities, initialData, definition);
         this.fileDescription = fileDescription;
         this.testDescription = testDescription;
+        assumeFalse(testDescription.equals("Unacknowledged findOneAndReplace with hint string on 4.4+ server"));
```

Comment:
Skipping these tests.  See downstream changes comment on JAVA-4275 for an explanation.